ansible/roles/selinux/tasks/main.yml

```diff
@@ -14,14 +14,26 @@
 # limitations under the License.
 
 - block:
-  - name: Disable SELinux
-    selinux:
-      configfile: '{{selinux_conf}}'
-      state: disabled
-      policy: targeted
-    register: status
+  - name: Check SELinux
+    stat:
+      path: "{{ selinux_conf }}"
+    register: stat_result
 
-  - name: Reboot
-    reboot:
-    when: status.reboot_required and reboot
+  - block:
+    - name: Ensure SELinux is set to disabled mode
```

Comment:
I didn't change the module name so if the PR is not merged, it will remain not fully qualified.  

The issue here is that the ansible.posix.selinux module does not work with certain OSs e.g. Rocky Linux 8, especially with our need for a more modern version (>3.6) of python when building Slurm.

My assumption is that if we always reboot (unless `reboot: false` which overrode the selinux module regardless) we won't have this issue.  A reboot during an image build should not be significant burden, especially when there's an expectation that it can happen some percentage of the time anyway.